src/resources/js/commerce/notice.js

```diff
@@ -0,0 +1,137 @@
+/* global tribe, jQuery */
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since TBD
+ *
+ * @type   {Object}
+ */
+tribe.tickets = tribe.tickets || {};
+
+/**
+ * Configures ET Tickets Commerce Object in the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type   {Object}
+ */
+tribe.tickets.commerce = tribe.tickets.commerce || {};
+
+/**
+ * Configures ET Tickets Commerce Object in the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type   {Object}
+ */
+tribe.tickets.commerce.notice = tribe.tickets.commerce.notice || {};
+
+/**
+ * Initializes in a Strict env the code that manages the plugin tickets commerce.
+ *
+ * @since TBD
+ *
+ * @param  {Object} $   jQuery
+ * @param  {Object} obj tribe.tickets.commerce
+ *
+ * @return {void}
+ */
+( function( $, obj ) {
+	'use strict';
+	const $document = $( document );
+
+	/*
+	 * Tickets Commerce Selectors.
+	 *
+	 * @since 5.1.9
+	 */
+	obj.selectors = {
+		hiddenElement: '.tribe-common-a11y-hidden',
+		item: '.tribe-tickets__commerce-checkout__notice',
+		content: '.tribe-tickets__commerce-checkout__notice__content',
+		title: '.tribe-tickets-notice__title',
```

Comment:
Is this something we can change? to follow the rest I'd say that `tribe-tickets__notice-title` would be the way to go.

Reply:
This is from the main notice template, so changing it would require updating snapshots: https://github.com/the-events-calendar/event-tickets/blob/5bd72e995c18dfb764821f31f944cdcdbde869db/src/views/components/notice.php#L36